R/sem.R

```diff
@@ -1044,46 +1038,57 @@ checkLavaanModel <- function(model, availableVars) {
 
     mutab$addColumnInfo(name = "lhs",      title = gettext("Variable"),   type = "string")
     mutab$addColumnInfo(name = "label",    title = "",                    type = "string")
-    mutab$addColumnInfo(name = "est",      title = gettext("Estimate"),   type = "number")
+    mutab$addColumnInfo(name = "est",      title = estTitle,   type = "number")
     mutab$addColumnInfo(name = "se",       title = gettext("Std. Error"), type = "number")
     mutab$addColumnInfo(name = "z",        title = gettext("z-value"),    type = "number")
     mutab$addColumnInfo(name = "pvalue",   title = gettext("p"),          type = "pvalue")
     mutab$addColumnInfo(name = "ci.lower", title = gettext("Lower"),      type = "number",
```

Comment:
`ciLower`

Reply:
not doing this, because the names of some columns in the lavaan output are also ci.lower and the find and replace messes it up...

Reply:
okay, just please leave a note in the merged PR saying "Frantisek notes that the code does not follow the JASP style guidelines" :p